packages/clay-color-picker/src/Custom.tsx

```diff
@@ -64,12 +65,12 @@ const RGBInput: React.FunctionComponent<IRGBInputProps> = ({
 						type="text"
 						value={inputValue}
 					/>
-					<label className="input-group-inset-item input-group-inset-item-before">
+					<ClayForm.Input.GroupInsetItem before tag="label">
```

Comment:
I went with tag because it wasn't necessarily a `div` or `label`. We also use it as a `span` for dropdown search. Since there was multiple elements possible, I figured it was best to put as a prop.

Reply:
Hmm, yeah it makes sense. Ok, I think we can go with that for now, it may slightly diminish this impression I have. Now that it will shorten the namespaces.